_posts/en/2021-05-18-problems-of-project-based-funding.md

```diff
@@ -0,0 +1,131 @@
+An IT team responsible for a product is in a constant struggle to re-adjust its work based on changing demands in addition to providing long-term costing estimates to these individual projects.
+This product team will spend a great deal of energy on planning expectations of work with other peer IT teams[^5] to be able to fulfil commitments it made to all these projects.
+
+<div style="display: block; width: inherit; text-align: center; line-height: 3;">
+    <a href="/ITStrategy/assets/images/project-planning-effect-on-product-teams-en.png" style="display: block">
+        <img src = "{{ site.baseurl }}/assets/images/project-planning-effect-on-product-teams-en.png"
+        alt = "This image depicts the cognitive load of multiple projects hitting a product team, and that product team needing to coordinate and plan with other IT teams (that are part of the value stream). Ultimately, the product team is being asked to provide costing and scheduling estimates for each individual projects. Providing this information is also work."
+        style="width: 800px; height: inherit;"
+        />
+    </a>
+    <small>Figure 1. Multiple projects hitting the product team, requiring them to coordinate with the other teams involved in the value stream</small>
+</div>
+
+When re-prioritization at the organization level occurs, a large-scale shift in demands occurs.
+The limited internal capacity of the IT organization needs to be re-adjusted to follow this new set of priorities.
+This usually results in A-Base funding re-allocation to fulfill pre-made commitments to the new priority projects  because the IT organization cannot wait for new funds to be released (funds from the programs funding these projects or released by TB subs).
+This leaves the IT organizations dipping into funds it currently has: either from existing program funding or [ARLU](https://www.oag-bvg.gc.ca/internet/English/parl_oag_200611_00_e_14968.html#ch0hd3d).
+The IT organization’s A-Base, funding its salary workforce meant to keep the lights on (operations), is then heavily involved in carrying the work until funding gets adjusted, HR processes for hiring new positions completes, and Procurement processes for acquiring services finishes.
+
+<div style="display: block; width: inherit; text-align: center; line-height: 3;">
+    <a href="/ITStrategy/assets/images/project-reprioritization-effects-en.png" style="display: block">
+        <img src = "{{ site.baseurl }}/assets/images/project-reprioritization-effects-en.png"
+        alt = "This image depicts the effects of re-prioritization has on IT teams. It involves having to re-do planning work, asking the A-Base funded teams to shift their attention to the new projects until new funds are made available to hire specialized staff and release the A-Base team from those priorities"
```

Comment:
As above I would suggest different alt text something like "This image has a clock at its centre. Around outside of the image of the clock there are a number of other images. At the 12 o'clock mark, there is an image that has a silhouette of a person and an icon to their right with the letters A, B, C in connected boxes and the words 'Reprioritization' underneath. At the 2 o'clock mark there is a silhouette of a person running down an inclined line with the words 'Product team re-plans' underneath.  At the 3 o'clock mark there is an image of a money back with a red x through it with the words 'Additional funds in transit' underneath. At the 4 o'clock mark there are 2 icons each showing a silhouette of a person carrying a heavy box with the words 'A-base teams fill the gaps' underneath. At the 6 o'clock mark there is an image of a stack of papers with a magnifying glass on top and the words 'Hiring process started' underneath. At the 8 o'clock mark there is an image of a car wrapped in a bow with a dollar sign above it and the words 'Additional funds arrive' underneath. At the 9 o'clock mark there is an image of three groups of people with the words 'Specialized people hired' underneath. At the 10 o'clock mark, there is an image of a stack of books and a person reading a book with the words 'Learning curve' underneath.